topic.tf

```diff
@@ -0,0 +1,29 @@
+resource "google_pubsub_topic" "topic" {
+  for_each = toset(var.topic.externally_managed ? [] : [var.topic.name])
+
+  labels  = local.labels
+  name    = "${var.team}__${var.topic.name}"
```

Comment:
*Playing devils advocate* and *Some thoughts*
Do we need the prefix to be the team? It is already included in the tags.
I think it would make more sense to be the product or label. It does make this PR more complicated though and would require more team specific modules 😞 
Example for label-platform:
- `rentaroof__new_subscribers_alert`
- `pararius__new_subscribers_alert`

Then again they could just include the label inside the schema of the topic e.g.
topic = `new_subscribers_alert` and schema has key `label`.
Then team prefix would make sense...

Anyways... **LGTM**

Reply:
I've pondering about adding an optional `application` part. Like: `<team>__<app>__<name>` / `<team>__<name>`.

Rationale: Label Platform will have Rentaroof specific resources, but Casco/Aanbod **might** as well.

And while typing this, probably "better" to make it even more generic: `<team>__<group>__<name>`, where `group` can be whatever `team` wants it to be.

Reply:
Taking things further, we could even disallow the usage of underscores in any of those parts. To prevent them from doing stuff like `name = "group__name"`. But, that's over-engineering again, at this point at least.

Reply:
or disallow double underscores, making our prefixes special in a way